suites/pacific/cephfs/tier-4_cephfs_generic.yaml

```diff
@@ -0,0 +1,150 @@
+#=======================================================================================================================
+# Tier-level: 4
```

Comment:
It is the last cephfs_generic component test which is being automated. Does it need a new suite file? Can it be added to any other existing suites considering the conf file used is the generic one?

Reply:
Added it tier-4_cephfs_recovery.yaml suite